### DISCUSSION TITLE
control the drone

### DISCUSSION THREAD

**Discussion by iore11** [bitcraze/discussions#1611] [Q&A] [answered]
Hello,

I have purchased the crazyflie 2.x drone and crazyflie PA, but there has been a problem with the drone not flying even though it is connected to cfclient. Also, everything has been installed. 

I feel like the instructions were a bit confusing for a beginner :(, I installed python, firmware and the driver and updated everything. And have purchased PA and the drone as I mentioned. But why is it still not flying? Did a propeller test and everything was fine. Except for an “error” that said something like (failed: src/stabilizer C:274) could it be the problem?

**Comment by knmcguire** [comment, marked as answer]
Hi!

We needs some more information about your problem. I'm quoting this line from the discussion rules:

> Provide as much information about your problem as you can. The [cfclients](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/)'s debug tab (found under 'Help'>'about') can already provide handy information about your development environment and your connected Crazyflie. Copy pasting the[ console tab's content](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/console_tab/) also helps a lot.

Thanks!

**Reply by knmcguire** [reply]
so, what actually help you out fixing your issue?
